tokens/src/core/global/other.json

```diff
@@ -4,5 +4,10 @@
     "$type": "percentage",
     "source": "$theme-color-interval",
     "$value": "8%"
+  },
+  "print-page-size": {
+    "source": "$print-page-size",
+    "$value": "a3",
+    "$type": "text"
```

Comment:
I agree with the previous use of the `dimension` type for the `print-page-size design` token. However, I believe this type contradicts the documented description of the dimension type provided by the [DTCG](https://second-editors-draft.tr.designtokens.org/format/#dimension).